contracts/Core/VoteManager.sol

```diff
@@ -56,6 +94,13 @@ contract VoteManager is Initializable, VoteStorage, StateManager, VoteManagerPar
         }
     }
 
+    /**
+     * @notice staker reveal the votes that they had committed to the protocol in the commit state.
+     * Stakers would only reveal the collections they have been allocated, the rest of their votes wont matter
```

Comment:
```
Stakers would only reveal the collections they have been allocated, the rest of their votes won't matter
```
This line is ambiguous.

Please mention that order of votes passed matter
And you will be only able to reveal what's assigned to you 
Values for others don't matter is correct, but only in tree, you cant pass them as votes

Also is possible 
Specify what staker pass in that Merkle tree struct